backend/webapi/Features/ClientLogs/Create.cs

```diff
@@ -0,0 +1,47 @@
+namespace Pidp.Features.ClientLogs;
+
+using FluentValidation;
+using Pidp.Data;
+using Pidp.Models;
+
+public class Create
+{
+    public class Command : ICommand
+    {
+        public string Message { get; set; } = string.Empty;
+
+        public LogLevel LogLevel { get; set; }
+
+        public string? AdditionalInformation { get; set; }
+    }
+
+    public class CommandValidator : AbstractValidator<Command>
+    {
+        public CommandValidator()
+        {
+            this.RuleFor(x => x.Message).NotEmpty();
+            this.RuleFor(x => x.LogLevel).NotEmpty();
```

Comment:
In PRIME and PIdP, our own `Enums` start from 1, generally. `LogLevel`, however, starts at 0, which corresponds to a logging level of "Trace". This validator unfortunately means that we could never create a `ClientLog` with a value of "Trace" since that is the default value of the `Enum` and `.NotEmpty()` means it cannot be the default value of the type.  This is fine as is, it is unlikely we would need to do "Trace" Client Logs; this is more for your own information.

Reply:
BTW I would also add the `.IsInEnum()` Validator for good measure.